cmd/sealer/cmd/cluster/scale-up.go

```diff
@@ -39,8 +38,8 @@ User can scale-up cluster by explicitly specifying host IP`
 
 var exampleForScaleUpCmd = `
 scale-up cluster:
-  sealer join --masters 192.168.0.1 --nodes 192.168.0.2 -p Sealer123
-  sealer join --masters 192.168.0.1-192.168.0.3 --nodes 192.168.0.4-192.168.0.6 -p Sealer123
+  sealer scale-up --masters 192.168.0.1 --nodes 192.168.0.2 -p Sealer123
```

Comment:
Are you find that, when you change the join.go or apply.go, you need change this file too. This is a bad design, you should use a common function for both command join, scale-up and apply. 

When should we refactor the code? The answer is NOW.